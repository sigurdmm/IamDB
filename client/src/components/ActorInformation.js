import React from 'react';
import PropTypes from 'prop-types';
import './InformationSection/index.less';

import InformationSection from './InformationSection';

const ActorInformation = ({ details }) => <InformationSection details={details}/>;

ActorInformation.propTypes = {
  details: PropTypes.shape({
    name: PropTypes.string.isRequired,
  }).isRequired,
};

export default ActorInformation;
