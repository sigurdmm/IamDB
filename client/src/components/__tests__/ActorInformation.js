import { render } from 'enzyme';

import ActorInformation from '../ActorInformation';

const mockedDetails = {
  thumbnails: {
    small: 'image',
  },
  name: 'title',
  popularity: 8.8,
};

describe('<ActorInformation/>', () => {
  it('should render properly', () => {
    const wrapper = render(<ActorInformation details={mockedDetails}/>);

    expect(wrapper).toMatchSnapshot();
  });
});
