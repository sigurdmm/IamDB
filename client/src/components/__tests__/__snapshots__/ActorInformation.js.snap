// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActorInformation/> should render properly 1`] = `
<div
  class="information__container"
>
  <div
    class="mediaimage"
  >
    <img
      class="image__item"
      src="image"
    />
  </div>
  <div
    class="information"
  >
    <div
      class="title"
    >
      <h2>
        title
      </h2>
    </div>
    <h3>
      ⭐️: 8.8
    </h3>
  </div>
</div>
`;
